hdf5.spec

```diff
@@ -141,8 +141,9 @@ HDF5 static libraries.
 Summary: HDF5 mpich libraries
 BuildRequires: mpich-devel
 Provides: %{name}-mpich2 = %{version}-%{release}
-Obsoletes: %{name}-mpich2 < 1.8.11-4
+Obsoletes: %{name}-mpich2 < %{version}-%{release}
```

Comment:
This should not be necessary.  RPM implies that `$package-$n` `Obsoletes $package < $n`.
```suggestion
```